Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml

```diff
@@ -264,7 +264,7 @@
     <!-- XHED: Vaccination Detail expiration Date -->
     <string name="expiration_date">Gültig bis %s, %s Uhr</string>
     <!-- XHED: Vaccination Detail expiration info -->
-    <string name="expiration_info">Wenn dies Ihr aktuell verwendetes Zertifikat ist, bemühen Sie sich bitte rechtzeitig darum, es zu erneuern. Ab 28 Tage vor Ablauf können Sie solche Zertifikate direkt über die App erneuern lassen. Sie finden die Option "Zertifikate erneuern" dann in Ihrer Zertifikatsübersicht unter der Kachel "Status-Nachweis". Sollte dies nicht Ihr aktuell verwendetes Zertifikat sein, muss es nicht verlängert werden und Sie müssen nichts weiter tun.</string>
+    <string name="expiration_info">Wenn dies Ihr aktuell verwendetes Zertifikat ist, bemühen Sie sich bitte rechtzeitig darum, es zu erneuern. Ab 28 Tage vor Ablauf können Sie solche Zertifikate direkt über die App erneuern lassen. Sie finden die Option \"Zertifikate erneuern\" dann in Ihrer Zertifikatsübersicht unter der Kachel \"Status-Nachweis\". Sollte dies nicht Ihr aktuell verwendetes Zertifikat sein, muss es nicht verlängert werden und Sie müssen nichts weiter tun.</string>
```

Comment:
I think the text for Expired and Expiring _Soon  should be the same and it should be :
```suggestion
    <string name="expiration_info">Wenn dies Ihr aktuell verwendetes Zertifikat ist, bemühen Sie sich bitte rechtzeitig darum, es zu erneuern. Ab 28 Tage vor Ablauf können Sie solche Zertifikate direkt über die App erneuern lassen. Sie finden die Option "Zertifikate erneuern" dann in Ihrer Zertifikatsübersicht unter der Kachel "Status-Nachweis".</string>
```

@thomasaugsten  can you confirm?